test/decompile/loadstore.txt

```diff
@@ -117,7 +117,7 @@ export function f(a:{ a:float, b:float }, b:{ a:ushort, b:long }) {
   f[0]:short;
   g[0]:int;
   g[0]:int@1;
-  h[1]:float@1;
+  h[1];
```

Comment:
no type here?

Reply:
Yup that's exactly what the change is intended to do. Rather than have types on all accesses, only have them once on the pointer decl.